.appveyor.yml

```
@@ -41,9 +41,13 @@ build_script:
   - scons platform=%GD_PLATFORM% target=%TARGET% tools=%TOOLS% %OPTIONS% %EXTRA_ARGS%
 
 after_build:
-  - 7z a godot_build.zip bin\*.exe
+  - git rev-parse --short=9 HEAD>VERSION_HASH.txt
+  - set /P VERSION_HASH=<VERSION_HASH.txt
```

```suggestion
  - git rev-parse --short=9 HEAD > VERSION_HASH.txt
  - set /P VERSION_HASH= < VERSION_HASH.txt
```
(Provided that this works on Windows - if not, keep as is.)